highcharts_excentis/highcharts/highcharts.py

```diff
@@ -348,12 +348,17 @@ def buildhtmlheader(self):
             opener.addheaders = [('User-Agent', 'Mozilla/5.0')]
 
             self.header_css = [
-                '<style>%s</style>' % opener.open(h).read() for h in self.CSSsource
+                '<style>' +
+                opener.open(source).read().decode('ASCII').replace('\n', '') +
+                '</style>' for source in self.CSSsource
             ]
 
             self.header_js = [
-                '<script type="text/javascript">%s</script>' % opener.open(h).read() for h in self.JSsource
+                '<script type="text/javascript">' +
+                opener.open(source).read().decode('ASCII').replace('\n', '') +
```

Comment:
Does the standard `decode()` or `decode('utf-8')` also work?

It is the preferred encoding.

```suggestion
                opener.open(source).read().decode('utf-8').replace('\n', '') +
```

Reply:
Yes, both seem to work! :)